_aggregations/metric/extended-stats.md

```diff
@@ -57,8 +92,13 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 }
 ```
 
-The `std_deviation_bounds` object provides a visual variance of the data with an interval of plus/minus two standard deviations from the mean.
-To set the standard deviation to a different value, say 3, set `sigma` to 3:
+## Defining bounds
+
+You can set the `std_deviation_bounds` plus-or-minus interval to any non-negative value using the `sigma` parameter.
+
+### Example: Defining bounds
+
+Set the `std_deviation_bounds` standard deviations to 3:
```

Comment:
```suggestion
Set the `std_deviation_bounds` standard deviations to `3`:
```

Reply:
It's up to you whether you want to keep this example. In general, users know how to pass parameters in a request.

Reply:
It's a short example, let's leave it in. I'll limit such examples in the future to ones that illustrate something interesting.